packages/cli/src/Server.ts

```diff
@@ -2204,7 +2205,7 @@ async function getExecutionsCount(countFilter: IDataObject): Promise<{ count: nu
 
 async function generateInstanceId(){
 	const encryptionKey = await UserSettings.getEncryptionKey();
-	const hash = encryptionKey ? createHash('sha256').update(encryptionKey).digest('hex') : undefined;
+	const hash = encryptionKey ? createHash('sha256').update(encryptionKey.slice(16)).digest('hex') : undefined;
```

Comment:
magic number... you should either add this as a constant to the top of the file, or maybe base this on the length of the encryption key
```suggestion
	const hash = encryptionKey ? createHash('sha256').update(encryptionKey.slice(encryptionKey.length / 2)).digest('hex') : undefined;
```